multichain-testing/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -103,21 +103,44 @@ const toOracleOfferId = (idx: number) => `oracle${idx + 1}-accept`;
 
 test.serial('oracles accept', async t => {
   const { oracleWds, retryUntilCondition, vstorageClient, wallets } = t.context;
+  const brands = await vstorageClient.queryData('published.agoricNames.brand');
+  const { Invitation } = Object.fromEntries(brands);
 
-  const instances = await vstorageClient.queryData(
-    'published.agoricNames.instance',
-  );
-  const instance = fromEntries(instances)[contractName];
+  // TODO export/import INVITATION_MAKERS_DESC
```

Comment:
should not be imported. the test should be declaring what it expects the value to be, not just that it matches something else. 

consider if someone changed the value in the other module. the test should fail and have to be updated to make clear that the API  changed.